make-release.sh

```diff
@@ -27,132 +67,185 @@ Usage: $0 --version [VERSION TO RELEASE]
 Example: $0 --version 7.25.2 -t
 
 Options: 
-  --trigger-release, -t  tag this release
-  --no-commit, -n        do not commit changes to branches
+  --trigger-release, -t   Tag this release
+  --nocommit, -n          Do not commit changes to git branches
+  --nopush                Do not push changes to git remote
+  -tmp, --use-tmp-dir     Use a fresh git clone in a temporary directory
 "
 }
 
-set -e
```

Comment:
we don't want to fail on error?

Reply:
Yes, the statement has been moved to line 25.